static/app/stores/groupStore.tsx

```diff
@@ -206,8 +237,8 @@ const storeConfig: GroupStoreDefinition = {
     return -1;
   },
 
-  addActivity(id, data, index = -1) {
-    const group = this.get(id);
```

Comment:
this function is mutating the group, this.get() now returns the wrong item to mutate so i'm switching it to just this.items.find